forge-gui/res/cardsfolder/upcoming/intruders_inquisition.txt

```diff
@@ -0,0 +1,9 @@
+Name:Intruder's Inquisition
+ManaCost:B R
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ DBDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature an opponent controls.
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | RememberTargets$ True | TgtPrompt$ Select target creature an opponent controls | AILogic$ PowerDmg | NumDmg$ ParentTargeted$CardPower | ExcessSVar$ Excess | DamageSource$ ParentTarget | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | ConditionCheckSVar$ Excess | ConditionSVarCompare$ GE1 | Defined$ RememberedController | NumCards$ 1 | DiscardValid$ Card.nonLand+cmcEQX | Mode$ TgtChoose | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup
+SVar:X:Count$ValidHand Card.RememberedController$GreatestCMC
```

Comment:
This isn't correct. I find it hard to make this work with to different targetedcontrollers

Reply:
just use ControlledBy to access full Defined power

Reply:
Any pointers ?

Reply:
`ControlledBy ParentTarget`

Reply:
Got it working this way thanks